docs/crd/application.giantswarm.io_appcatalog.yaml

```
@@ -0,0 +1,95 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  name: appcatalogs.application.giantswarm.io
+spec:
+  group: application.giantswarm.io
+  names:
+    kind: AppCatalog
+    plural: appcatalogs
+    singular: appcatalog
+  scope: Cluster
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: |
+        An AppCatalog defines a collection of apps to install in one or several clusters.
```

```suggestion
        An AppCatalog defines a collection of apps that can be installed in one or more clusters.
```

This reads to me as if all apps need to be installed when actually you don't need to install any.

We've talked before about having an "app stack" to have a higher level abstraction for installing multiple apps. So I'd like it to be clearer that the catalog is just a grouping of apps that are stored together.